openstack/barbican/values.yaml

```diff
@@ -307,3 +307,14 @@ hsm:
 utils:
   trust_bundle:
     enabled: true
+
+# barbican nanny
+barbican_nanny:
+  replicas: 1
+  enabled: false
+  # debug mode - no script run in the containers, just a sleep loop
+  debug: false
+  # run the nanny every n minutes
+  interval: 1
```

Comment:
imho better: Sleep time in minutes between runs

Reply:
its 1min, updating secrets could be done any time know. Should i increase it ?

Reply:
I was talking about the comment. It's not true that the nanny runs every minute if it takes 5min to run. It will then only run every 6min. Therefore, I think replacing the comment with "Sleep time in minutes between runs" would be more accurate.